renku/core/workflow/plan_factory.py

```diff
@@ -79,19 +79,21 @@ def __init__(
         self.no_output_detection = no_output_detection
 
         if not command_line:
-            raise errors.UsageError("Command line can not be empty.")
+            raise errors.UsageError("Command line can not be empty. Please specify a command to execute.")
 
         if not directory:
             directory = os.getcwd()
         self.directory = Path(directory)
         if not self.directory.exists():
-            raise errors.UsageError("Directory must exist.")
+            raise errors.UsageError(
+                "Current directory doesn't exist. Please make sure you are inside an existing directory."
+            )
 
         if not working_dir:
             working_dir = os.getcwd()
         self.working_dir = Path(working_dir)
         if not self.working_dir.exists():
-            raise errors.UsageError("Working Directory must exist.")
+            raise errors.UsageError("Repository path doesn't exist. Make sure you are inside a Renku repository.")
```

Comment:
It's good to print the directory in the message: `f"Repository path doesn't exist: {self.working_dir} ..."